Calculation.py

```diff
@@ -0,0 +1,18 @@
+import output as op
+import spouseOrParentShares as sps
+
+def calculate(amount,sons,daughters,mother,father,wife):
```

Comment:
You should install a linter to lint and format the code properly. There are multiple options, the most commonly used are `ruff`, `black` etc. Have a look at: 
https://github.com/vintasoftware/python-linters-and-code-analysis